src/battle_animation.h

```diff
@@ -76,6 +76,13 @@ class BattleAnimation : public Sprite {
 	 **/
 	void SetNoOffset(bool nooffset);
 
+	/**
+	 * Set if the animation is coming from a battle animation sprite
+	 *
+	 * @param battleanimationsprite
+	 **/
+	void SetBattleAnimationSprite(bool battleanimationsprite);
```

Comment:
Lets give this a better name, like "SetOverwriteScreenEffects". From the point of view of the BattleAnimation class it's not about whether it's a sprite or not. It's if the screen effects get overwritten even frame.

It just so happens that BattleAnimationSprite use case uses this feature.

Reply:
Did the renaming.